libs/components/modals/src/lib/modules/modal/modal.component.ts

```diff
@@ -117,6 +135,7 @@ export class SkyModalComponent implements AfterViewInit {
     private windowRef: SkyAppWindowRef,
     private componentAdapter: SkyModalComponentAdapterService,
     private coreAdapter: SkyCoreAdapterService,
+    private mediaQueryService: SkyResizeObserverMediaQueryService,
```

Comment:
Since we're removing `SkyMediaQueryService` completely from the constructor, what's the point of overwriting its provider in the `providers` array?

Reply:
I could be wrong - but I believe that is being done to make sure all children components of the modal are looking at the `SkyResizeObserverMediaQueryService` instead of the app one for their own size breakpoints.